README.md

```diff
@@ -250,9 +252,24 @@ secrets:
 ```
 And make it accessible to tentaclio by setting the environmental variable `TENTACLIO__SECRETS_FILE`. The actual name of each url is for traceability and has no effect in the functionality.
 
-Alternatively you can run `curl https://raw.githubusercontent.com/octoenergy/tentaclio/master/extras/init_tentaclio.sh` to create a secrets file in `~/.tentaclio.yml` and 
+Alternatively you can run `curl https://raw.githubusercontent.com/octoenergy/tentaclio/master/extras/init_tentaclio.sh` to create a secrets file in `~/.tentaclio.yml` and
 automatically configure your environment.
 
+## Configuring access to google drive.
+Google drive support is _experimental_ and should be used at your own risk. Aslo due to google drive itself it's rather slow.
+
+1. Get the credentials.
+First we need a credentials file in order to be able to generate tokens. The easiest way to do this is by going to [this example](https://developers.google.com/drive/api/v3/quickstart/python),
+click on enable drive api, and download the generated file.
+
+2. Generate token file 
+
+```
+pipenv install tentaclio && pipenv run -m tentaclio google-token generate
+```
+This will open a browser with a google auth page, log in and accept the authorisation request.
+The token file has been saved in a default location. You can also configure this via the env variable `TENTACLIO__GOOGLE_DRIVE_TOKEN_FILE`
```

Comment:
Is the original credentials JSON required after the token file is created? I think no since there's a refresh key - could add a line here specifying that

Reply:
you're right no need for the credentials.json, I'll add that up.